app/models/institution.ts

```diff
@@ -30,7 +30,7 @@ export default class Institution extends OsfModel {
     @computed('assets', 'id')
     get logoUrl(this: Institution): string {
         let assetsUrl = '';
-        if (this.assets !== null) {
+        if (this.assets !== null && this.assets !== undefined) {
```

Comment:
A truthy check is probably good enough here?

Reply:
Sure, if that's what you're into. Updated.